client/components/token-field/index.jsx

```diff
@@ -183,6 +183,12 @@ var TokenField = React.createClass( {
 			} );
 		}
 
+		if ( autoCapitalize ) {
+			Object.assign( props, {
+				autoCapitalize: autoCapitalize
+			} );
```

Comment:
>Optional, but we can add a little sugar and do:

There's also a critical difference between the two that makes @gwwar's suggestion preferable, which is that the object spread syntax assigns values into a new object. In the original code, `Object.assign` will mutate props directly, which could have unintended consequences. If kept with current syntax, you'd want to pass an empty object as the first argument so that `props` is not mutated.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign